configuration/configuration_test.go

```diff
@@ -0,0 +1,831 @@
+package configuration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"strings"
+
+	"github.com/almighty/almighty-core/configuration"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/spf13/cast"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	tDefaultConfigurationFile           = "config.yaml"
+	tEnvironmentVariableNameConfigFile  = "ALMIGHTY_CONFIG_FILE_PATH"
+	tEnvironmentVariableValueConfigFile = "../config.yaml"
+	tInvalidConfigFilePath              = "../invalid_config.yaml"
+	tTestConfigFilePath                 = "../test/data/multiple_config.yaml"
+
+	// The following will be used to set env variables for testing.
+	tTestEnvString         = "ALMIGHTH_CONFIG_VALUE"
```

Comment:
Thanks for pointing out @kwk :) , I've fixed it, though it wouldn't have caused a negative impact since I'm consuming the variable anyway. 

Really appreciate the time you are putting in to help me out with this PR.